### PR TITLE
Fix(general-ledger): Add empty state message

### PR DESCRIPTION
**Pull request by joshuahedlund**
Also moved fiscal year <select> out of the grid footer and deprecated the footer so if ledger is empty user can still select a different year

### CODE REVIEW

**Comment by jniles** [comment]
@joshuahedlund, this is fantastic!  I'll pull the code and give you some feedback this evening.  It looks excellent though!  Really clean!

File: client/src/modules/general-ledger/general-ledger.html [hunk 1]
```diff
@@ -87,7 +87,7 @@
 
         <div
           id="generalLedger-grid"
-          class="grid-full-height"
+          class="grid-full-height-with-filters"
```
Comment (jniles) [inline client/src/modules/general-ledger/general-ledger.html:90]: 👍

File: client/src/modules/general-ledger/general-ledger.html [hunk 2]
```diff
@@ -96,13 +96,24 @@
...
+            empty-state="GeneralLedgerCtrl.gridOptions.data.length===0"
+            empty-state-message="TABLE.COLUMNS.EMPTY"
             error-state="GeneralLedgerCtrl.hasError" >
           </bh-grid-loading-indicator>
 
         </div>
+        <div class="ui-grid-cell-contents">
+          <select
+            name="fiscalYear"
+            style="cursor: pointer"
```
Comment (jniles) [inline client/src/modules/general-ledger/general-ledger.html:108]: `cursor: pointer` is a nice touch!

File: client/src/modules/general-ledger/grid.footer.html [hunk 3]
```diff
@@ -1,11 +0,0 @@
-<div class="ui-grid-cell-contents">
```
Comment (jniles) [inline client/src/modules/general-ledger/grid.footer.html:1]: Thanks for cleaning up this old file 👍

**Comment by jniles** [comment]
Excellent, well done!  It all looks good to me.

bors r+

**Comment by joshuahedlund** [comment]
Great, glad to be of service!

File: client/src/modules/general-ledger/general-ledger.html (hunk 2 above)
Comment (joshuahedlund) [reply client/src/modules/general-ledger/general-ledger.html:108]: To be fair, I can't take credit for that one as I copied it from grid.footer.html :smiley:

**Pull request merged**